### PR TITLE
Add encodedTokenAndUrl for tokens

### PR DESCRIPTION
**Pull request by yuting-liu**
Added the missing field `encodedTokenAndUrl` for installation tokens.

### CODE REVIEW

**Comment by yuting-liu** [comment]
I cannot add Rishi as the reviewer in this PR.
Let me slack him directly.

**Review by vsinghal13 — APPROVED** [review]

**Review by maimaisie — COMMENTED** [review]
Any testing done? Also should the unit test verify this is correctly set?

File: sumologic/resource_sumologic_token_test.go [hunk 1]
```diff
@@ -180,11 +180,13 @@ resource "sumologic_token" "test" {
 func testAccCheckTokenAttributes(name string) resource.TestCheckFunc {
 	return func(s *terraform.State) error {
 		f := resource.ComposeTestCheckFunc(
+			resource.TestCheckResourceAttrSet(name, "id"),
 			resource.TestCheckResourceAttrSet(name, "name"),
 			resource.TestCheckResourceAttrSet(name, "description"),
 			resource.TestCheckResourceAttrSet(name, "status"),
 			resource.TestCheckResourceAttrSet(name, "type"),
 			resource.TestCheckResourceAttrSet(name, "version"),
+			resource.TestCheckResourceAttrSet(name, "encoded_token_and_url"),
```
Comment (yuting-liu) [inline sumologic/resource_sumologic_token_test.go:189]: This method is called for both create and update. There we added the check for `encoded_token_and_url`.
I'm also testing this locally

**Comment by yuting-liu** [comment]
The created token:
```
$ terraform show
# sumologic_token.test_token:
resource "sumologic_token" "test_token" {
    description           = "Testing resource sumologic_token"
    encoded_token_and_url = "U1VNT05DbDZFR3NqRGtyaVVGczdadWdmZGlGR2FPU1FodHRwczovL25pdGUtZXZlbnRzLnN1bW9sb2dpYy5uZXQ="
    id                    = "000000000001C26C"
    name                  = "testToken"
    status                = "Active"
    type                  = "CollectorRegistration"
    version               = 0
}
```

The updated token:
```
yuting@yliu-mac plugins $ terraform show
# sumologic_token.test_token:
resource "sumologic_token" "test_token" {
    description           = "Testing resource sumologic_token updated"
    encoded_token_and_url = "U1VNT05DbDZFR3NqRGtyaVVGczdadWdmZGlGR2FPU1FodHRwczovL25pdGUtZXZlbnRzLnN1bW9sb2dpYy5uZXQ="
    id                    = "000000000001C26C"
    name                  = "testToken updated"
    status                = "Inactive"
    type                  = "CollectorRegistration"
    version               = 1
}
```

File: sumologic/resource_sumologic_token_test.go (hunk 1 above)
Comment (maimaisie) [reply sumologic/resource_sumologic_token_test.go:189]: great thanks

**Review by maimaisie — APPROVED** [review]

**Review by rishiyemme — APPROVED** [review]

**Pull request merged**